libs/sdk-ui-filters/src/AttributeFilter/utils.ts

```diff
@@ -131,10 +131,15 @@ export function validateAttributeFilterProps(props: IAttributeFilterBaseProps) {
     if (selectionMode === "single" && filter) {
         invariant(
             isPositiveAttributeFilter(filter) && attributeElementsCount(filterAttributeElements(filter)) < 2,
-            "Provided 'filter' property is not compatible with given selectionMode. It needs to be positive filter with max one item selected in attribute elements",
+            "Provided 'filter' property is not compatible with given single selection mode. It needs to be positive filter with max one item selected in attribute elements",
         );
     }
 
+    invariant(
+        selectionMode !== "single" || (selectionMode === "single" && isEmpty(parentFilters)),
+        "Parent filtering can not be used together with single selection mode. Use only one of these properties at the same time.",
+    );
```

Comment:
What happens if I have filter on KD, change MD filter context via API so this invariant will be triggered (parent filtered filter set to single or add parent filter to SSF) an then access such dashboard?